src/rlx_util.erl

```diff
@@ -33,30 +34,41 @@
          indent/1,
          render/2,
          load_file/3,
-         template_files/0]).
+         template_files/0,
+         sh/1]).
 
 -export([os_type/1]).
 
 -define(ONE_LEVEL_INDENT, "     ").
 
 -include("rlx_log.hrl").
 
-relx_sasl_vsn() ->
+is_sasl_gte() ->
+    %% default check is for sasl 3.5 and above
+    %% version 3.5 of sasl has systools with changes for relx
+    %% related to `make_script', `make_tar' and the extended start script
+    is_sasl_gte({{3, 5, 0}, {[], []}}).
+
+is_sasl_gte(Version) ->
     application:load(sasl),
     case application:get_key(sasl, vsn) of
         {ok, SaslVsn} ->
-            %% TODO: change to actual version of sasl with systools changes
-            parse_vsn(SaslVsn) > {{infinity, infinity, infinity}, [], []};
+            not(parsed_vsn_lt(parse_vsn(SaslVsn), Version));
         _ ->
             false
     end.
 
 %% parses a semver into a tuple {{Major, Minor, Patch}, {PreRelease, Build}}
 -spec parse_vsn(string()) -> {{integer(), integer(), integer()}, {string(), string()}}.
 parse_vsn(Vsn) ->
-    case re:run(Vsn, "^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)"
-                "(-(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*)?"
+    case re:run(Vsn, "^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.?(0|[1-9][0-9]*)?"
```

Comment:
Yea, I had sort of not cared if that was the case because it screwed up the matching. but now I realize maybe I can just replace `3` with `4`... not sure how I missed that before.

Reply:
Yea, that works fine. Updating the commit.